libr/core/project.c

```diff
@@ -706,6 +713,11 @@ R_API bool r_core_project_save(RCore *core, const char *prjName) {
 	if (scr_null) {
 		r_config_set_i (core->config, "scr.null", true);
 	}
+	if (!ret) {
```

Comment:
It should also reset it if oldPrjName is NULL. Or is it not allowed to do r_config_set (core->config, "prj.name", 0)? In that case it should probably be set to "".